daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -319,40 +319,39 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments)
+    descr("Parse data to a DFDL infoset")
     helpWidth(76)
 
-    val schema = opt[URI]("schema", argName = "file", descr = "the annotated DFDL schema to use to create the parser.")(fileResourceURIConverter)
-    val rootNS = opt[RefQName]("root", argName = "node", descr = "the root element of the XML file to use.  An optional namespace may be provided. This needs to be one of the top-level elements of the DFDL schema defined with --schema. Requires --schema. If not supplied uses the first element of the first schema")
-    val path = opt[String](argName = "path", descr = "path to the node to create parser.")
-    val parser = opt[File](short = 'P', argName = "file", descr = "use a previously saved parser.")
-    val output = opt[String](argName = "file", descr = "write output to a given file. If not given or is -, output is written to stdout.")
-    val validate: ScallopOption[ValidationMode.Type] = opt[ValidationMode.Type](short = 'V', default = Some(ValidationMode.Off), argName = "mode", descr = "the validation mode. 'on', 'limited', 'off', or a validator plugin name.")
-    val vars = props[String]('D', keyName = "variable", valueName = "value", descr = "variables to be used when parsing. An optional namespace may be provided.")
-    val tunables = props[String]('T', keyName = "tunable", valueName = "value", descr = "daffodil tunable to be used when parsing.")
-    val config = opt[File](short = 'c', argName = "file", descr = "path to file containing configuration items.")
-    val infosetType = opt[InfosetType.Type](short = 'I', argName = "infoset_type", descr = "infoset type to output. Must be one of 'xml', 'scala-xml', 'json', 'jdom', 'w3cdom', 'sax', or 'null'.", default = Some(InfosetType.XML))
-    val stream = toggle(noshort = true, default = Some(false), descrYes = "when left over data exists, parse again with remaining data, separating infosets by a NUL character", descrNo = "stop after the first parse, even if left over data exists")
-    val infile = trailArg[String](required = false, descr = "input file to parse. If not specified, or a value of -, reads from stdin.")
+    val config = opt[File](short = 'c', argName = "file", descr = "Path to XML file containing configuration items")
+    val vars = props[String]('D', keyName = "variable", valueName = "value", descr = "Variables to be used when parsing. An optional namespace may be provided.")
+    val infosetType = opt[InfosetType.Type](short = 'I', argName = "infoset_type", descr = "Infoset type to output. Use 'xml', 'scala-xml', 'json', 'jdom', 'w3cdom', 'sax', or 'null'.", default = Some(InfosetType.XML))
+    val output = opt[String](argName = "file", descr = "Output file to write infoset to. If not given or is -, infoset is written to stdout.")
+    val parser = opt[File](short = 'P', argName = "file", descr = "Use a previously saved parser")
+    val path = opt[String](argName = "path", descr = "Path from root element to node from which to start parsing")
```

Comment:
I wonder if we should hide this path option by adding `hidden = true`? Daffodil only supports `/`, so cluttering the help output with this option that isn't really an option probably doesn't add anything and just makes the help output more verbose. And by hiding it, people can still use it so it won't break backwards compatibility.

Reply:
I second this change.